config/v2/config.go

```diff
@@ -8,101 +8,112 @@ import (
 
 	v1 "github.com/chanzuckerberg/fogg/config/v1"
 	"github.com/chanzuckerberg/fogg/errs"
+	"gopkg.in/yaml.v2"
 )
 
+//ReadConfig take a byte array as input and outputs a json or yaml config file
 func ReadConfig(b []byte) (*Config, error) {
+	var e error
 	c := &Config{
 		Docker: false,
 	}
-	e := json.Unmarshal(b, c)
 
-	return c, errs.WrapUser(e, "unable to parse config")
+	if v1.IsJSON(b) {
+		e = json.Unmarshal(b, c)
+	} else {
+		e = yaml.Unmarshal(b, c)
+	}
+	if e != nil {
```

Comment:
can probably do without this conditional and just `return c, errs.WrapUser(e, ...)`

Reply:
I tried that earlier but since c is initialized to a Config, the change fails all of the nil tests.